package.json

```diff
@@ -167,23 +167,21 @@
 		"@types/validator": "^13.7.1",
 		"@types/webpack-env": "^1.16.3",
 		"@types/wordpress__api-fetch": "^3.2.4",
-		"@types/wordpress__block-editor": "^6.0.5",
```

Comment:
We are removing some of those type packages because they augment or redeclare the `@wordpress/data` types, which is in direct conflict with the new way we're consuming `@wordpress/data` types - directly from the package.